test/scenarios/ynLSD.spec.sol

```diff
@@ -1,78 +1,217 @@
 // SPDX-License-Identifier: BSD-3-Clause
 pragma solidity ^0.8.24;
 
-import { IntegrationBaseTest } from "test/integration/IntegrationBaseTest.sol";
-import { Invariants } from "test/scenarios/Invariants.sol";
-import { IERC20 } from "lib/openzeppelin-contracts/contracts/interfaces/IERC20.sol";
-import { TestAssetUtils } from "test/utils/TestAssetUtils.sol";
+import {Invariants} from "./Invariants.sol";
```

Comment:
the identation in this file is messed up, it uses 8 spaces intead of 4

Reply:
hmm not sure what you're referring to here

Reply:
I had this issue as well @johnnyonline... I had to change my cursor/vscode settings to use spaces, instead of tabs. Might be the cause of this one for you too